index.html

```diff
@@ -475,64 +475,67 @@
+    given information that is relevant to them and that will increase the [=user=]'s
+    autonomy, as much as possible in a manner that they will comprehend and at the right
+    moment. This is almost never when the [=user=] is trying to do something else such as
     read a page or activate a feature. The duty of honesty goes well beyond that of
-    transparency that dominates legacy privacy regimes. Unlike with transparency, honesty
-    cannot get away with hiding relevant information in complex out-of-band legal notices
-    no more than it can rely on overly cursory information provided in a consent dialog.
+    transparency that is often included in older privacy regimes. Unlike transparency, honesty
+    can't hide relevant information in complex out-of-band legal notices and it can't rely on
+    very short summaries provided in a consent dialog.
   </dd>
   <dt><dfn>Duty of Loyalty</dfn></dt>
   <dd>
-    Because of the special [=fiduciary relationship=] that obtains between [=user=] and
-    [=user agent=], the latter is held to be loyal to the former in all situations, up to and
-    including in preference to the [=user agent=]'s implementer. When a [=user agent=] carries
-    out [=processing=] that is not directly in the [=user=]'s interest but rather benefits
-    another entity such as its implementer, including by piggybacking on [=processing=] that
-    may be in the user's interest, that behaviour is known as <dfn>self-dealing</dfn>.
-    [=Self-dealing=] is always [=inappropriate=]. Loyalty is the avoidance of [=self-dealing=].
+    Because the [=user agent=] is a [=trustworthy agent=], it is held to be loyal to the
+    [=user=] in all situations, including in preference to the [=user agent=]'s implementer.
+    When a [=user agent=] carries out [=processing=] that is not directly in the [=user=]'s
+    interest but rather benefits another entity such as the implementer that behaviour is
+    known as <dfn>self-dealing</dfn>. Behaviour is [=self-dealing=] even if it is done at the
+    same time as [=processing=] that is in the [=user=]'s interest. [=Self-dealing=] is always
+    [=inappropriate=]. Loyalty is the avoidance of [=self-dealing=].
   </dd>
 </dl>
 
 These duties ensure the [=user agent=] will <em>care</em> for the user. It is important to
-note that there is a subtle difference between care and <dfn>data paternalism</dfn> which
-is that the latter purports to help in part by removing agency ("<em>don't worry about it,
+note that there is a subtle difference between care and <dfn>data paternalism</dfn>.
+[=Data paternalism=] claims to help in part by removing agency ("<em>don't worry about it,
 so long as your data is with us it's safe, you don't need to know what we do with it, it's
-all good because we're good people</em>") whereas care aims to support people by enhancing
+all good because we're good people</em>") whereas [=care=] aims to support people by enhancing
```

Comment:
There's no `<dfn>` for "care".
```suggestion
all good because we're good people</em>") whereas care aims to support people by enhancing
```